train_model.py

```diff
@@ -230,13 +231,24 @@ def main():
     )
 
     # Instantiate model + trainer
+    if args.eval:
+        use_distributed_sampler = False
+    else:
+        use_distributed_sampler = True
```

Comment:
I would rather keep the warning for this, but still leave the possibility to run eval on multi-gpu. If you run with batch size 1 the problem with distributed sampler goes away, and this is nice to be able to do with very large models.

Reply:
okay fair point, I'll remove these lines.